yinstruments/pdu/README.md

```diff
@@ -6,9 +6,19 @@ see several classes that help organization and drive connections to different
 PDUs. The PDU class is the parent class to the Netbooter and Lindy. All of these 
 classes have the following callable functions: get_status, reboot, off, and on.
 
+SETUP: There are two things that you will need to do before the commands provided will work:
+
+    1. make python enviornment and go into enviornment
+        --> make env
+        --> source .venv/bin/activate
+
+    2. install snmp package into your enviornment
+        --> sudo apt update
+        --> sudo apt install snmp snmpd
```

Comment:
This applies for CLI or packages usage, so you might want to move it before the package/CLI subsections.